src/g_game.c

```diff
@@ -4502,151 +4502,150 @@ void doomprintf(player_t *player, msg_category_t category, const char *s, ...)
   M_BindBool("monster_infighting", &default_monster_infighting, &monster_infighting,
              true, ss_none, wad_yes,
-             "1 to enable monsters fighting against each other when provoked");
+             "Monsters fight each other when provoked");
   M_BindBool("monster_backing", &default_monster_backing, &monster_backing,
              false, ss_none, wad_yes,
-             "1 to enable monsters backing away from targets");
+             "Ranged monsters back away from melee targets");
   M_BindBool("monster_avoid_hazards", &default_monster_avoid_hazards, &monster_avoid_hazards,
              true, ss_none, wad_yes,
-             "1 to enable monsters to intelligently avoid hazards");
+             "Monsters avoid hazards such as crushing ceilings");
   M_BindBool("monkeys", &default_monkeys, &monkeys, false, ss_none, wad_yes,
-             "1 to enable monsters to move up/down steep stairs");
+             "Monsters move up/down steep stairs");
   M_BindBool("monster_friction", &default_monster_friction, &monster_friction,
              true, ss_none, wad_yes,
-             "1 to enable monsters to be affected by friction");
+             "Monsters are affected by friction modifiers");
   M_BindBool("help_friends", &default_help_friends, &help_friends,
-             false, ss_none, wad_yes, "1 to enable monsters to help dying friends");
+             false, ss_none, wad_yes, "Monsters prefer targets of injured allies");
   M_BindNum("friend_distance", &default_distfriend, &distfriend,
-            128, 0, 999, ss_none, wad_yes, "Distance friends stay away");
+            128, 0, 999, ss_none, wad_yes, "Minimum distance that friends keep between each other");
   M_BindBool("dog_jumping", &default_dog_jumping, &dog_jumping,
-             true, ss_none, wad_yes, "1 to enable dogs to jump");
+             true, ss_none, wad_yes, "Dogs are able to jump down from high ledges");
 }
 
 void G_BindCompVariables(void)
 {
   M_BindNum("default_complevel", &default_complevel, NULL,
             CL_MBF21, CL_VANILLA, CL_MBF21, ss_comp, wad_no,
-            "0 = Vanilla, 1 = Boom, 2 = MBF, 3 = MBF21");
+            "Default compatibility level (0 = Vanilla; 1 = Boom; 2 = MBF; 3 = MBF21)");
   M_BindBool("autostrafe50", &autostrafe50, NULL, false, ss_comp, wad_no,
-             "1 to enable auto strafe50");
+             "Automatic strafe50 (SR50)");
   M_BindBool("strictmode", &default_strictmode, &strictmode,
-             false, ss_comp, wad_no, "1 to enable strict mode");
+             false, ss_comp, wad_no, "Strict mode");
   M_BindBool("hangsolid", &hangsolid, NULL, false, ss_comp, wad_no,
-             "1 to walk under solid hanging bodies");
+             "Enable walking under solid hanging bodies");
   M_BindBool("blockmapfix", &blockmapfix, NULL, false, ss_comp, wad_no,
-             "1 to enable blockmap bug fix");
+             "Fix blockmap bug");
```

Comment:
It's not really a blockmap bug, but rather a restriction, or not?

Reply:
We call it "Improved Hit Detection" in the menu. Doomwiki article says "blockmap bug" is popular https://doomwiki.org/wiki/Flawed_collision_detection

Reply:
What about `Fix blockmap bug (improves hit detection)`?

Reply:
Yes, fine.